exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -66,9 +74,44 @@ impl JavaServiceRuntime {
         }
     }
 
-    /// Returns internal service proxy.
-    pub fn service_proxy(&self) -> ServiceProxy {
-        self.service_proxy.clone()
+    /// Loads an artifact from the specified location involving verification of the artifact.
+    /// Returns an unique service artifact identifier that must be specified in subsequent
+    /// operations with it.
+    pub fn load_artifact(&self, artifact_uri: &str) -> String {
```

Comment:
This operation does not seem to be needed till it has meaningful applications (in subsequent tasks). I'd keep it as it's already added, but please extend the description of the following tasks to include tests for this operation.

Reply:
@skletsun :arrow_up_small:

Reply:
Updated tasks, going to cover it with tests within ECR-2979

Reply:
Good, thanks! Is there anything needed in fakes?

Reply:
Perhaps we'd need entities that (1) simulates artifact loading failure and (2) that is just loaded well (for **sunny day** and **artifact is already loaded** scenarios)